web_external/ImagesGallery/Listing/ImagesPagingPane.js

```diff
@@ -65,52 +71,55 @@ isic.views.ImagesPagingPane = Backbone.View.extend({
+            .prop('disabled', !this.images.hasPreviousPage());
+        this.$('#isic-images-paging-seek-next, #isic-images-paging-seek-last')
+            .prop('disabled', !this.images.hasNextPage());
 
         // Show the relevant explanatory label
-        this.$('.detailLabel').hide();
+        this.$('#isic-images-paging-label>*').hide();
         var labelElement;
         if (hasFilters && hasPaging) {
-            labelElement = this.$('#isic-images-hasFiltersAndPaging');
+            labelElement = this.$('#isic-images-paging-label-hasFiltersAndPaging');
         } else if (hasFilters) {
-            labelElement = this.$('#isic-images-hasFilters');
+            labelElement = this.$('#isic-images-paging-label-hasFilters');
         } else if (hasPaging) {
-            labelElement = this.$('#isic-images-hasPaging');
+            labelElement = this.$('#isic-images-paging-label-hasPaging');
         } else {
-            labelElement = this.$('#isic-images-noPagingOrFilters');
+            labelElement = this.$('#isic-images-paging-label-noPagingOrFilters');
         }
         labelElement.show();
 
         // Update the values in the label
-        labelElement.find('span.overview')
+        labelElement.find('.isic-images-paging-label-total')
           .text(this.completeFacets.total);
-        labelElement.find('span.filteredSet')
+        labelElement.find('.isic-images-paging-label-filtered')
           .text(this.filteredFacets.total);
         if (hasPaging) {
             // Use base 1 for the page text labels
             var startImageNum = this.images._currentOffset() + 1;
             var endImageNum = startImageNum + this.images.length - 1;
-            labelElement.find('span.page')
+            labelElement.find('.isic-images-paging-label-page')
                 .text(startImageNum + ' - ' + endImageNum);
         } else {
-            labelElement.find('span.page')
+            labelElement.find('.isic-images-paging-label-page')
                 .text(this.images.length);
         }
 
         // Move the paging bar
-        this.$('#isic-images-pagingBars .page')
+        this.$('#isic-images-paging-bar-pageTotal')
             .css('left', _.bind(function () {
                 return (this.images._currentOffset() / this.filteredFacets.total) * 100 + '%';
             }, this))
             .width(_.bind(function () {
                 return (this.images.length / this.filteredFacets.total) * 100 + '%';
             }, this));
 
-        this.$('#isic-images-download-zip').toggleClass(
+        this.$('#isic-images-paging-downloadZip').prop(
             'disabled', this.filteredFacets.total === 0);
+
+        // Any just-disabled buttons will no longer trigger 'mouseleave' or 'focusout' events, so
+        // any still-active tooltips on those buttons must be manually hidden
+        this.$('[data-toggle="tooltip"][disabled]').tooltip('hide');
```

Comment:
Perhaps related, it would be nice for tooltips to show on the disabled buttons. We should see if adding tooltips to wrapper elements allows them to show for the disabled buttons.